fe/fe-core/src/main/java/com/starrocks/sql/analyzer/ShowStmtAnalyzer.java

```diff
@@ -182,6 +183,19 @@ public Void visitShowRoutineLoadStatement(ShowRoutineLoadStmt node, ConnectConte
             return null;
         }
 
+        @Override
+        public Void visitShowRoutineLoadTaskStatement(ShowRoutineLoadTaskStmt node, ConnectContext context) {
+            String dbName = node.getDbFullName();
+            dbName = getDatabaseName(dbName, context);
+            node.setDbFullName(dbName);
+            try {
+                node.checkJobNameExpr();
+            } catch (AnalysisException e) {
+                throw new SemanticException("analysis show routine load task error:", e);
```

Comment:
SemanticException can not be passed with cause exception. better to add a log here and then throw SemanticException